eng/emitters/scripts/Generate-APIView-CodeFile.ps1

```diff
@@ -1,7 +1,7 @@
 param (
   [Parameter(mandatory = $true)]
   $ArtifactPath,
-  $NpmDevopsFeedRegistry = "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-js/npm/registry/"
+  $NpmDevopsFeedRegistry = "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-js@local/npm/registry/"
```

Comment:
Do you know if this is case sensitive? I've always seen it as `@Local`

Reply:
Documentation refers both @local and @Local. I will update it to @Local.